src/lib/states/parallel.js

```diff
@@ -58,6 +52,20 @@ class Parallel extends State {
     };
   }
 
+  catchError(err) {
+    // Throw error if no catch
+    if (!this.state.Catch) {
+      const error = err instanceof Error ? err : new Error(err.cause);
+      throw error;
+    }
+    // TODO: Implement ErrorEquals
```

Comment:
I thought you implemented it?

Reply:
Indeed, just forgot to add it to Parallel state !
Now added